game_patch/graphics/gr_font.cpp

```diff
@@ -250,6 +250,7 @@ GrNewFont::GrNewFont(std::string_view name) :
     static std::pair<int, int> win_1252_char_ranges[]{
         {0x20, 0x7E},
         {0x8C, 0x8C},
+        {0x95, 0x95},
```

Comment:
will it work if someone plays with resolution 1024x768 or less? Do we always use TTF font for that header? If not how separator will look like in that case?

Reply:
No it will not render for the small hud I think. But does it really matter?

Reply:
Well, I sometimes use that resolution myself to test things in window. It can't crash, that's obvious. But it would be nice to also look good. If it looks okayish I guess we can leave it like this. Ultimately if we want to introduce more characters in the future (e.g. unicode?) we may want to fully stop using builtin fonts, then the problem will solve itself